odml/section.py

```diff
@@ -192,6 +192,12 @@ def append(self, obj):
         else:
             raise ValueError("Can only append sections and properties")
 
+    def append(self, *obj_tuple):
+        """append Sections or Properties"""
+        ll = len(obj_tuple)
+        for i in list(range(0,ll)):
+            self.__append(obj_tuple[i])
```

Comment:
@jgrewe Thanks for the review. I tend to write as if it is C. Updates will follow soon.